README.md

```diff
@@ -5,20 +5,13 @@ It contains a reference implementation of the Markowitz portfolio optimization
 problem and the data used in the paper. Please note that the tickers of the
 stocks have been obfuscated to comply with the data provider's terms of use.
 
-## Installation
-
-Please start any experiment with
-
-```bash
-make install
-```
-
-to replicate the virtual environment we have defined in 'requirements.txt'.
-
 ## Experiments
 
 Please run all experiments using
 
 ```bash
 make experiments
 ```
+
+Before the experiments start the virtual environment defined in 'requirements.txt'
+is replicated locally.
```

Comment:
```suggestion
Before running the experiments, install the dependencies listed in `requirements.txt`.
```

Reply:
The dependencies are installed automatically, no action is required